homeassistant/components/homepilot/__init__.py

```diff
@@ -0,0 +1,80 @@
+"""The Rademacher HomePilot integration."""
+
+import asyncio
+from datetime import timedelta
+import logging
+
+from pyhomepilot.api import HomePilotAPI
+from pyhomepilot.auth import Auth
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import aiohttp_client
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+
+from .const import DOMAIN, SCAN_INTERVAL
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORMS = ["cover"]
+
+
+async def async_setup(hass: HomeAssistant, config: dict):
+    """Set up the Rademacher HomePilot component."""
+    hass.data.setdefault(DOMAIN, {})
+
+    return True
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
+    """Set up Rademacher HomePilot from a config entry."""
+    session = aiohttp_client.async_create_clientsession(hass)
+    auth = Auth(session, entry.data["host"], entry.data.get("password"))
```

Comment:
would suggest to use constants from `homeassistant.const` like `CONF_HOST` and `CONF_PASSWORD`